src/python/zigopt/authentication/result.py

```diff
@@ -1,27 +1,22 @@
 # Copyright © 2022 Intel Corporation
 #
 # SPDX-License-Identifier: Apache License 2.0
-def authentication_result(
-  token=None,
-  client=None,
-  user=None,
-  permission=None,
-  membership=None,
-  authenticated_from_email_link=None,
-  authenticated_public_cert=None,
-  authorization_response=None,
-  session_expiration=None,
-  organization=None,
-):
-  return {
-    "token": token,
-    "client": client,
-    "user": user,
-    "permission": permission,
-    "membership": membership,
-    "authenticated_from_email_link": authenticated_from_email_link,
-    "authenticated_public_cert": authenticated_public_cert,
```

Comment:
dropped intentionally due to no SAML/ext-authz?

Reply:
yes, vulture complained that it was unused